src/schema.js

```diff
@@ -1,19 +1,39 @@
+import { get } from 'axios'
 import {
   GraphQLObjectType,
   GraphQLString,
-  GraphQLSchema
-  // GraphQLInt,
-  // GraphQLList,
-  // GraphQLNonNull
+  GraphQLSchema,
+  GraphQLInt,
+  GraphQLList
 } from 'graphql'
 
+const ItemType = new GraphQLObjectType({
+  name: 'Item',
+  description: 'An item from the HN API',
+  fields: () => ({
+    by: { type: GraphQLString },
+    descendants: { type: GraphQLInt },
+    id: { type: GraphQLInt },
+    kids: { type: new GraphQLList(GraphQLInt) },
+    score: { type: GraphQLInt },
+    text: { type: GraphQLString },
+    time: { type: GraphQLInt },
+    title: { type: GraphQLString },
+    type: { type: GraphQLString },
+    url: { type: GraphQLString }
```

Comment:
each field should have it's own resolver, so you can query them separately.
see here: https://github.com/jgcmarins/graphql-ufc-api/blob/master/src/schema.js

Reply:
Fixed on d3af8f1! Ty